airbyte-integrations/connectors/source-mailchimp/source_mailchimp/manifest.yaml

```diff
@@ -0,0 +1,366 @@
+version: 0.52.0
+type: DeclarativeSource
+
+check:
+  type: CheckStream
+  # TODO: how to make simple ping ?
```

Comment:
i've tried to, but unfortunately, streams to check should be declared in `streams` list

Reply:
It seems like [this line](https://github.com/airbytehq/airbyte/blob/f67938993e687a4b2ce2fed9ff81315f7f92e2cd/airbyte-cdk/python/airbyte_cdk/sources/declarative/checks/check_stream.py#L31) assumes the stream we test is indeed part of the catalog. 

For this specific case, are we losing information using a stream instead of a ping? I think we have some sources that queries an endpoint to retrieve the scopes of the auth token and prints the missing scopes for example which is clearly added value. What is the added value we would be losing in this case?

Reply:
Not in this case, I'll remove todo to avoid confusion